src/dodal/devices/oav/oav_to_redis_forwarder.py

```diff
@@ -78,13 +79,13 @@ def __init__(
         """
         self.counter = epics_signal_r(int, f"{prefix}CAM:ArrayCounter_RBV")
 
-        self._sources = DeviceVector(
+        self.sources = DeviceVector(
             {
                 Source.ROI.value: OAVSource(f"{prefix}MJPG:", "roi"),
                 Source.FULL_SCREEN.value: OAVSource(f"{prefix}XTAL:", "fullscreen"),
             }
         )
-        self.selected_source = soft_signal_rw(Source)
+        self.selected_source = soft_signal_rw(int)
```

Comment:
Should: I think we should probably make this a string enum then. It will tidy up the log messages etc. too

Reply:
Unfortunately the integer is required for the use of `DeviceVector`